### DISCUSSION TITLE
Asked for a school ID although I selected a verified school-issued email.

### DISCUSSION THREAD

**Discussion by yuuahp** [community/community#56456] [GitHub Education]
### Hi everyone,

I selected the school-issued email to renew my academic status but was prompted to upload a dated proof and my applications keep being rejected for the profile issue.
However, [FAQ](https://github.com/orgs/community/discussions/17814) said that that happens if I selected my personal email or an untrusted email.
I applied last year and two years ago with the same email and was approved normally.
![image](https://github.com/community/community/assets/53546237/05d8a3d6-fdb4-4b30-90a5-d82546653971)
![image](https://github.com/community/community/assets/53546237/2778a614-f06b-44d7-836c-9f6d33f3fd9d)

Here is what I tried so far:
- Wrote my full name on my profile.
- Removed and re-added the email.
- Sent application at the school.
- Searched for the same issue. ("have completed your GitHub Profile")

**Comment by samus-aran** [comment]
Hi @yuuahp 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
